samples/batch/transcription-enabled-storage/Connector/StorageConnector.cs

```diff
@@ -119,17 +129,24 @@ public async Task WriteTextFileToBlobAsync(string content, string containerName,
         {
             log.LogInformation($"Writing file {fileName} to container {containerName}.");
             var container = BlobServiceClient.GetBlobContainerClient(containerName);
-            var blockBlob = container.GetBlobClient(fileName);
+            var blockBlobClient = container.GetBlobClient(fileName);
+
+            if (await blockBlobClient.ExistsAsync().ConfigureAwait(false))
+            {
+                log.LogError($"File {fileName} already exists in container {containerName}. Returning.");
+                return;
+            }
 
             using (var stream = new MemoryStream(Encoding.UTF8.GetBytes(content)))
             {
```

Comment:
instead of using  stream, I would prefer to use UpdateTextAsync() to write the file with one-shot.  For streaming, it is possible that the file is only partially written.

Reply:
Update/UploadTextAsync was part of microsoft.windowsazure.storage I think, at least I don't have it in azure.storage.blobs anymore